### ISSUE TITLE
Testing Continuity, AirDrop, Sidecar

### ISSUE THREAD

**Issue by H3LLC47** [huyhoang8398/x1c7-hackintosh-20R1#2]
I tested Continuity, AirDrop, and SideCar on an iPad Air 2, iPad Air 4, and an iPad 7th Gen. None of them worked. Maybe a separate Kext would fix it but in the current iteration of the EFI they don't work. Just thought I'd let you know since you don't have the equipment to test it.

**Comment by huyhoang8398** [comment]
Thank @H3LLC47 , I will try to look into it

**Comment by H3LLC47** [comment]
> Thank @H3LLC47 , I will try to look into it

Great, I have been looking at this as well because I use all of those features quite a bit

**Comment by ziqi1029** [comment]
Are there any new updates?

**Comment by H3LLC47** [comment]
> 

No luck yet, I have tried a couple of kexts.

**Comment by ziqi1029** [comment]
On my 20R1 laptop, I didn't make any changes of this EFI, the bluetooth can turn on but no device is found.

**Comment by huyhoang8398** [comment]
For Airdrop, I dont think it is possible with our Wifi card, since the moderator of itlwm said it won't be supported.
And for sidecar, I will borrow an iPad from my friend next week and have it test. I will inform you if it works

**Comment by ziqi1029** [comment]
> For Airdrop, I dont think it is possible with our Wifi card, since the moderator of itlwm said it won't be supported. And for sidecar, I will borrow an iPad from my friend next week and have it test. I will inform you if it works

This EFI  on the MacOS 10.15.6 was successfully installed with the EFI, today.

**Comment by H3LLC47** [comment]
Yeah, I have tried multiple things. I think the WiFi card is the cause. But it still doesn't explain the reason I cant use SideCar with a cable to the machine. I guess it's possible that the USBs aren't mapped correctly.
